### PR TITLE
feat: added product page

### PR DESCRIPTION
**Pull request by Creator-singh**
added product listing page
product_page

### CODE REVIEW

File: styles/product.css [hunk 1]
```diff
@@ -0,0 +1,100 @@
+.product-container{
+    display: flex;
```
Comment (aniketsingh13) [inline styles/product.css:2]: you can use utility for this

File: styles/product.css [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    top: 8rem;
+    left: 0;
+    width: 25rem;
+    min-width: 20rem;
+    height: calc(100vh - 8rem);
+    overflow: auto;
+}
+
+.filter-title{
+    display: flex;
```
Comment (aniketsingh13) [inline styles/product.css:17]: here also

**Review by aniketsingh13 — APPROVED** [review]
consider the changes other thing good to go

**Review by Rohitprasad83 — APPROVED** [review]
LGTM

**Pull request merged**